c/src/point_one/polaris/polaris.c

```diff
@@ -64,7 +64,7 @@ static int SendPOSTRequest(PolarisContext_t* context, const char* endpoint_url,
 
 static int GetHTTPResponse(PolarisContext_t* context);
 
-static void CloseSocket(PolarisContext_t* context);
+static void CloseSocket(PolarisContext_t* context, int destroy_context);
 
 #ifdef POLARIS_USE_TLS
```

Comment:
This may trigger a param unused warning when the TLS usage isn't defined. Not a huge issue.

Reply:
True, will fix.

Reply:
I tested and it does not.